iac-extensions/docker/src/main/java/org/sonar/iac/docker/parser/grammar/DockerGrammar.java

```diff
@@ -75,4 +81,13 @@ public List<SyntaxToken> ARGUMENTS() {
       )
     );
   }
+
+  public StopSignalTree STOPSIGNAL() {
+    return b.<StopSignalTree>nonterminal(DockerLexicalGrammar.STOPSIGNAL).is(
+      f.stopsignal(
+        b.token(DockerKeyword.STOPSIGNAL),
+        b.token(STRING_LITERAL)
```

Comment:
Are multiple signals also allowed?

Reply:
I tried, multiple signal is not possible in one `STOPSIGNAL` instruction.